package/helm/openmrs/values.yaml

```diff
@@ -28,13 +28,24 @@ config:
   MODULE_WEB_ADMIN: false
   OPENELIS_HOST: openelis
   OPENELIS_PORT: '8052'
+  MAIL_TRANSPORT_PROTOCOL: 'smtps'
+  MAIL_SMTP_AUTH: true
+  MAIL_SMTP_STARTTLS_ENABLE: true
+  MAIL_SMTP_SSL_ENABLE: true
+  MAIL_DEBUG: false
+  SEND_MAIL: false
+  MAIL_FROM: 'noreply@mybahmni.in'
```

Comment:
Since this is going to be specific to the implementer's domain, this can also be passed from the deploy workflow.

Reply:
keep the default values in alignment to .env

Reply:
@mohan-13 @N0-man, done with the changes, please review once